src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/CrossCaseTypeSearchRequest.java

```diff
@@ -43,13 +44,15 @@
     private final ElasticsearchRequest elasticsearchRequest;
     private final boolean multiCaseTypeSearch;
     private final List<String> aliasFields = new ArrayList<>();
+    private boolean consolidationQuery;
```

Comment:
```suggestion
    private boolean isAnyCaseRequest;
```

Reply:
Furthermore, is this new property needed? I can't see it used

Reply:
DONE removed.